subworkflows/local/prepare_genome.nf

```diff
@@ -130,7 +130,7 @@ workflow PREPARE_GENOME {
     if (prepare_tool_index == 'bwa') {
         if (params.bwa_index) {
             if (params.bwa_index.endsWith('.tar.gz')) {
-                ch_bwa_index = UNTAR_BWA_INDEX ( params.bwa_index ).untar
+                ch_bwa_index = UNTAR_BWA_INDEX ( [], params.bwa_index ).untar.map{it -> it[1]}
```

Comment:
```suggestion
                ch_bwa_index = UNTAR_BWA_INDEX ( [], params.bwa_index ).untar.map{ it[1] }
```

Reply:
Addressed in [33d3bdd](https://github.com/nf-core/chipseq/pull/271/commits/33d3bdda056cc66bd6c9070bc0abf670e9c6c229)